### PR TITLE
An application that creates a hazelcast client to probe the status of a hazelcast cluster

### PR DESCRIPTION
**Pull request by sli-cs**

### CODE REVIEW

File: src/main/scala/io/callstats/HcProbe.scala [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+class HcStatusProbe {
+
+  @GET
+  @Path("ping")
+  def getServerTime(): String = {
+    "received ping at " + new Date().toString()
+  }
+
+  @GET
+  @JSONP
```
Comment (eljasala) [inline src/main/scala/io/callstats/HcProbe.scala]: Do we actually need JSONP?

File: src/main/scala/io/callstats/HcProbe.scala [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+object HcProbe {
+
+  def main(args: Array[String]): Unit = {
+
+    // the "HC_ADDRS" can be comma separated 'ip:port' tuples indicate the hazelcast node addresses
+    // if it is not defined, we'll just crash
+    var hcClusterAddrEnv = sys.env.get("HC_ADDRS")
+    if (!hcClusterAddrEnv.isDefined) {
+      sys.error("environment variable HC_ADDRS not set!")
+      sys.exit(-1)
```
Comment (eljasala) [inline src/main/scala/io/callstats/HcProbe.scala:113]: You should return after this to make sure nothing else gets executed (e.g. if exit failed for some reason)

**Review by eljasala — COMMENTED** [review]
What's the minimum amount of memory this is going to require to run?

File: src/main/scala/io/callstats/HcProbe.scala (hunk 1 above)
Comment (sli-cs) [reply src/main/scala/io/callstats/HcProbe.scala]: good point. we don't need it.

File: src/main/scala/io/callstats/HcProbe.scala (hunk 2 above)
Comment (sli-cs) [reply src/main/scala/io/callstats/HcProbe.scala:113]: 'return' added now.

**Comment by sli-cs** [comment]
I changed the dockerfile cmd to start the application. Now it will use 384M heapsize at maximum.

**Review by eljasala — APPROVED** [review]
LGTM

**Pull request merged**